src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -525,22 +525,14 @@ public String toString() {
 			result.append(", ");
 		}
 		result.append("Audio Codec: ").append(getAudioCodec());
-		if (getSampleFrequency() != null) {
-			result.append(", Sample Frequency: ").append(getSampleFrequency());
-		}
-		if (getBitRate() != 0) {
-			result.append(", Bitrate: ").append(getBitRate());
-		}
-		if (getBitsperSample() != 0 && getBitsperSample() != 16 ) {
+		result.append(", Sample Frequency: ").append(getSampleFrequency());
+		result.append(", Bitrate: ").append(getBitRate());
+		if (getBitsperSample() != 16 ) {
 			result.append(", Bits per Sample: ").append(getBitsperSample());
 		}
 
-		if (getAudioProperties() != null) {
-			result.append(", ").append(getAudioProperties());
```

Comment:
@Nadahar Since i was the one that make many of them, and you complained, i just deleted them.
Now i really don't bother, so do you want this `null` and the next `0` condition test or not ?